assets/scss/pdf.scss

```diff
@@ -533,11 +545,3 @@ hr {
     padding-top: 20px;
   }
 }
```

Comment:
@jjrobothamho 
Do you know why this is red?
Do you know why git is complaining about this?